PendingReleaseNotes.md

```diff
@@ -9,6 +9,7 @@
 - Ceph OSDs will be automatically updated by the operator when there is a change to the operator version or when the OSD configuration changes. See the [OSD upgrade notes](Documentation/upgrade-patch.md#object-storage-daemons-osds).
 - Rook Ceph block storage provisioner can now correctly create erasure coded block images. See [Advanced Example: Erasure Coded Block Storage](Documentation/block.md#advanced-example-erasure-coded-block-storage) for an example usage.
 - [Network File System (NFS)](https://github.com/nfs-ganesha/nfs-ganesha/wiki) is now supported by Rook with a new operator to deploy and manage this widely used server. NFS servers can be automatically deployed by creating an instance of the new `nfsservers.nfs.rook.io` custom resource. See the [NFS server user guide](Documentation/nfs.md) to get started with NFS.
+- The minimum version of Kubernetes supported by Rook changed from `1.7` to `1.8`.
```

Comment:
I would consider referencing this in the breaking changes section too.  In that section, you can mention that users of 1.7 should consider upgrading their k8s cluster (maybe even link to docs to help with that if you can find a good link?).

Reply:
@jbw976 Update docs as per your comment.

Reply:
looks great, thanks!